js/index.js

```diff
@@ -230,7 +287,11 @@ function CovidChart({covidData, regions, selectedTypes, useLogScale}) {
     }
     const labels = covidData.labels
       .slice(covidSchema.dateStartColumn)
-      .map(formatDateLabel);
+      .map(formatDateLabel)
+      .filter(x=> {
+        // if(!startDate) { return x }
```

Comment:
What do you think remove commented code? If don't needed I think we can remove.
```suggestion
```

Reply:
I was doing a flow testing by commenting it and forgot to remove before push. Apologies, i will update this once the code splitting case is resolved.

Reply:
Resolved this is https://github.com/trekhleb/covid-19/pull/30/commits/176d9fcb73fb239c389ee6bcb8711f32b4124c80